cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -119,6 +124,27 @@ def test_mobile_field_available(self):
         self.assertEqual(settings_fields["display_name"], "Mobile Course Available")
         self.assertEqual(settings_fields["deprecated"], True)
 
+    @ddt.data(
+        (False, False, True),
+        (True, False, False),
+        (True, True, True),
+        (False, True, True)
+    )
+    @ddt.unpack
+    def test_discussion_fields_available(self, is_pages_and_resources_enabled,
```

Comment:
@saadyousafarbi From what I understand the fields will be hidden only in case the ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND is True and OVERRIDE_DISCUSSION_LEGACY_SETTINGS_FLAG is False. Is that the expected behaviour? I thought the fields needed to be hidden based on boths flags. Or am I missing something?

Reply:
The fields will be hidden when the `ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND` is **True**.
If for some reason we want to revert to showing the settings back again in the legacy experience, we will set the `OVERRIDE_DISCUSSION_LEGACY_SETTINGS_FLAG` to **True**.